scripts/user-actions/add.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+console.log('--- loading add countries and capital');
+
+const addCountriesCapital= (object) => {
+
+};
+
+const addCountryHandler= () => {
+    debugger;
+    const userInput = prompt('add a new country:');
+    if(userInput === null || userInput === 'number'){
+        return;
+    }
+    const userinput1 = prompt('add the capital of the country')
+    if(userInput1 === null || userInput1 === 'number'){
+        return;
+    }
+    const message = `"${userInput}" "${userinput1}" has been added:`
+    console.log(message)
+
+};
```

Comment:
**you wrote some variables  misspelling and need to user inputs store in the shared global variable then call the display handler function to show them**

> I write some code in yours should work if you have a question about them let me know.


// I do not know why you added this function
/*
const addCountriesCapital = (object) => {

};
*/

const addCountryHandler = () => {
    debugger;
    const userInput = prompt('add a new country:');
    if (userInput === null || userInput === '') {
        return;
    }
    const userInput1 = prompt('add the capital of the country')
    if (userInput1 === null || userInput1 === '') {
        return;
    }

    // store input in global variable
    countries[userInput] = userInput1;

    const message = `Country "${userInput}" and its capital "${userInput1}" has been added:`
    console.log(message)

    // call display handler to show message

    displayHandler(message, countries);


};